docs/content/modules/player-lifecycle-feature-module.md

```diff
@@ -95,7 +103,52 @@ public static class OneTimeInitialization
 }
 ```
 
-### When is a player entity deleted?
+##### 3. (Optional) Manually request player creation
+
+If `AutoRequestPlayerCreation` is set to false, you must manually call `RequestPlayerCreation` in the `SendCreatePlayerRequestSystem`:
+```csharp
+var playerCreationSystem = World.GetExistingManager<SendCreatePlayerRequestSystem>();
+playerCreationSystem.RequestPlayerCreation();
+```
+
+## How to pass arbitrary data into a player creation request
```

Comment:
Maybe: "How to use arbitrary data in the player creation loop" or something like that. This covers it end to end really